gazebo_plugins/src/gazebo_ros_vacuum_gripper.cpp

```diff
@@ -192,23 +212,24 @@ void GazeboRosVacuumGripper::UpdateChild()
       math::Pose link_pose = links[j]->GetWorldPose();
       math::Pose diff = parent_pose - link_pose;
       double norm = diff.pos.GetLength();
-      if (norm < 0.05) {
+      if (norm <= max_distance_) {
         links[j]->SetLinearAccel(link_->GetWorldLinearAccel());
-        links[j]->SetAngularAccel(link_->GetWorldAngularAccel());
+        links[j]->SetAngularAccel(math::Vector3(0, 0, 0));
```

Comment:
I'm not sure if ignoring totally the angular acceleration and set it to 0 is the best option here. @scpeters , @hsu could you comment on this change please?